code/react/base/.eslintrc.cjs

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  plugins: ["@typescript-eslint", "solid"],
+  plugins: ["@typescript-eslint", "react"],
```

Comment:
The `react/base` eslint plugin was not changed from `solid` to `react`, so I'm changing it here.